run.sh

```diff
@@ -0,0 +1 @@
+npm run watch:dev
```

Comment:
@gutorc92, the purpose of having a separate `run.sh` is to have a logic and decide, based on a environment variable, if the application should be run in debug mode or not.

Reply:
The env variable DEBUG can be set to control the level of debug. ex: debug=express:* will show just messages from express.